_posts/2020-01-02-documenting-rcpp-packages.md

```diff
@@ -25,40 +25,36 @@ and have it appear in the package reference manual.
 
 This vignette demonstrates four different ways to document Rcpp functions and classes in R
 packages. We will create an R package from scratch, add some very basic functionality and show how
-Roxygen2 blocks appear in the final documentation. Throughout the vignette, we will focus on R
-and Rcpp to make our job easier. There are a number of popular add-on packages helping with 
-package development (_e.g._ `devtools`, `remotes`, `usethis` as of January 2020) but our focus here
-is on core packages.  The additional helpers may be reviewed on a follow-up vignette.  In case
-Rcpp is not yet installed, do
+Roxygen2 blocks appear in the final documentation. Throughout the vignette, we will focus on base R,
+Rcpp and roxygen2 with no additional dependencies. There are a number of popular add-on packages
+helping with package development (_e.g._ `devtools`, `remotes`, `usethis` as of January 2020), and
+these additional helpers may be reviewed in a follow-up vignette.  In case
+Rcpp or roxygen2 are not yet installed, do
 
 
 {% highlight r %}
-install.packages("Rcpp")
+install.packages(c("Rcpp","roxygen2"))
 {% endhighlight %}
 
 ### Create a package
 
 We begin by initializing a package skeleton. The following functions create a new directory and
-establish a barebones structure for an R package with Roxygen2 and Rcpp support:
+establish a barebones structure for an R package with Rcpp support:
 
 
 {% highlight r %}
-Rcpp.package.skeleton("~/test/testpkg")
+Rcpp::Rcpp.package.skeleton("testpkg", path="~/test/")
 {% endhighlight %}
 
 Using your favorite text editor, create a new file `~/test/testpkg/R/zzz.R` and add the following
 lines to it:
 
 
 {% highlight r %}
-#' @useDynLib testpkg
```

Comment:
Not needed since skeleton() already places `useDynLib()` in `NAMESPACE`. This particular `roxygen` directive will have no effect.

Reply:
Agreed.